pallets/governance/src/application.rs

```diff
@@ -134,6 +146,7 @@ pub fn accept_application<T: crate::Config>(application_id: u32) -> DispatchResu
     });
 
     // Pay the application cost back to the applicant
+    // TODO: should this value be used?
```

Comment:
Not really, it's a guard for a negative imbalance, which has no use here

Reply:
Shouldn't we store the Application cost as one of these "imbalances" intead of a plain number, so the total issuance doesn't change while the Application is open?

If it's not possible, is there another way to do something like this?

Also, btw, there's and open task to direct this burns to the treasury.  [CHAIN-60](https://linear.app/renlabs-dev/issue/CHAIN-60/send-fees-to-treasury-instead-of-burning)

Reply:
Iirc it doesnt implement the codecs needed. We could do something similar manually but im not a fan of this idea

Reply:
Btw, I **think** if was used like that, the replay bug would have been catched.

Reply:
> but im not a fan of this idea

why tho?